doc/subsystems/logging/logger.rst

```diff
@@ -335,6 +342,43 @@ the message), and severity level. Once all backends are iterated, the message
 is considered processed by the logger, but the message may still be in use by a
 backend.
 
+.. _logger_strings:
+
+Logging strings
+===============
+
+Logger stores the address of a log message string argument passed to it. Because
+a string variable argument could be transient, allocated on the stack, or
+modifiable, logger provides a mechanism and a dedicated buffer pool to hold
+copies of strings.  The buffer size and count is configurable
+(see :option:`CONFIG_LOG_STRDUP_MAX_STRING` and
+:option:`CONFIG_LOG_STRDUP_BUF_COUNT`).  
```

Comment:
checkpatch doesn't like the trailing spaces on this line... (https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/22096/5/tests)